src/accessors/ILogMessage.ts

```diff
@@ -0,0 +1,9 @@
+import { ILogMessageSeverity } from './LogMessageSeverity';
+/**
+ * Message which will be passed to a UI (either in a log or in the application's UI)
+ */
+export interface ILogMessage {
```

Comment:
@graywolf336 I'd propose to have a `console` for tracing purposes as it is usual as a globally accessible object. In addition, I believe that the Rocketlets need to have a way to communicate to the user (`Toastr` or alike)